packages/discord.js/typings/index.d.ts

```diff
@@ -568,17 +568,15 @@ export abstract class CommandInteraction<Cached extends CacheType = CacheType> e
   public deferReply(
     options: InteractionDeferReplyOptions & { withResponse: true },
   ): Promise<InteractionCallbackResponse>;
-  public deferReply(options?: InteractionDeferReplyOptions): Promise<InteractionResponse<BooleanCache<Cached>>>;
+  public deferReply(options?: InteractionDeferReplyOptions): Promise<undefined>;
```

Comment:
Shouldn't this be `InteractionCallbackResponse | undefined` since typing wise `withResponse` could be `boolean` and in runtime `true`?
Same goes for the others.